docs/specs/output.yml

```diff
@@ -614,3 +614,29 @@ outputs:
           }
       ]
     }
+---
+# Relocate error file path, git urls based on source map
+repos:
+  https://github.com/relocate/source-file:
+    - files:
+        docfx.yml: |
+          sourceMap: a/sourceMap.json
```

Comment:
Will the extensibility team be OK with this config file update?

Reply:
yes, I checked with @TianqiZhang and we agreed to use a dedicated file over yaml header for this. The file format is the same as the current mapping file produced from `ecma2yml`.